### PR TITLE
Fix Dependency Vulnerability

### LINKED ISSUE #10: Fix security issue by updating decompress-zip
Got a warning corresponding to this security issue:
https://www.npmjs.com/advisories/777

From the issue description:
> "For decompress-zip 0.3.x upgrade to 0.3.2 or later."

### LINKED ISSUE #163: Remove dependency on an achived package node-mksnapshot
There was a vulnerability in mksnapshot and a PR is opened https://github.com/electron-archive/node-mksnapshot/pull/11, however I'm not sure you all maintain it as it's in the electron-archive org.  I want to avoid forking this as well as mksnapshot to fix issue.  Any way you could either merge that PR or update this repo to remove that dep?

### PR DESCRIPTION
**Pull request by AstroGD**
decompress-zip has a vulnerability with high risk (Arbitrary File overwrite)
To fix this an update is mandatory

fixes #10
fixes https://github.com/electron/asar/issues/163

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "rimraf": "~2.1.4"
   },
   "dependencies": {
-    "decompress-zip": "0.3.2",
+    "decompress-zip": "^0.3.2",
```
Comment (AstroGD) [inline package.json]: This change is made so that upcoming updates >= 0.3.2 <0.4.0 will automatically be installed

**Comment by nucleardreamer** [comment]
I was just coming to this repo to make this exact same PR. Thanks for doing it @AstroGD !

**Comment by y0x** [comment]
Same here, but it seems that this package is no longer maintained?!

**Comment by jameskerr** [comment]
I need this fix as well. Thanks for the work to update it. Hope it gets merged soon.

**Comment by e2jk** [comment]
@zeke , @kevinsawicki , is there any hope to get this PR merged for this inactive package?
It is currently an indirect dependency of electron-packager (electron-packager > asar > mksnapshot > decompress-zip), all projects using electron-packager now have a "high severity vulnerability" being detected by `npm audit`.
(Note that @bizob2828 also logged an issue against asar to remove the dependency leading to this)

**Comment by bizob2828** [comment]
Not sure if anyone cares, but I took the time to fix this in a fork.  I didn't publish to npm as I hope this will eventually get resolved but you can just ```npm install Contrast-Security-OSS/asar#v0.14.7```

**Comment by AstroGD** [comment]
@bizob2828 Thanks for the effort. Ill try using your version of asar as long as there is no official fix. Did you create a PR for your fixed version at the asar Repo?

**Comment by bizob2828** [comment]
@AstroGD so i messed this up a bit. since I'm not publishing on npm i didn't compile the coffeescript, standby

**Comment by AstroGD** [comment]
It seems this PR is no longer needed as there was an update on this repo, that updated the dependencies.
(Im referring to this commit: 90d09561b7200487a12ce5f5825dfe44f5d6e299 )

**Pull request closed without merging**